src/rpc/blockchain.cpp

```diff
@@ -2699,6 +2699,91 @@ UniValue CreateUTXOSnapshot(
     return result;
 }
 
+static RPCHelpMan loadtxoutset()
+{
+    return RPCHelpMan{
+        "loadtxoutset",
+        "Load the serialized UTXO set from disk.",
+        {
+            {"path",
+                RPCArg::Type::STR,
+                RPCArg::Optional::NO,
+                "path to the snapshot file. If relative, will be prefixed by datadir."},
+        },
+        RPCResult{
+            RPCResult::Type::ELISION, "", "",
+        },
+        RPCExamples{
+            HelpExampleCli("loadtxoutset", "utxo.dat")
+        },
+        [&](const RPCHelpMan& self, const JSONRPCRequest& request) -> UniValue
+{
+    fs::path path = fs::PathFromString(request.params[0].get_str());
+    if (path.is_relative()) {
+        path = gArgs.GetDataDirNet() / path;
```

Comment:
In commit "rpc: add loadtxoutset" (82a8744f1384f07dc30aaa6339b0b8fb32dc3af0)

Would be good to simplify and remove gArgs reference:

```diff
--- a/src/rpc/blockchain.cpp
+++ b/src/rpc/blockchain.cpp
@@ -2718,10 +2718,8 @@ static RPCHelpMan loadtxoutset()
         },
         [&](const RPCHelpMan& self, const JSONRPCRequest& request) -> UniValue
 {
-    fs::path path = fs::PathFromString(request.params[0].get_str());
-    if (path.is_relative()) {
-        path = gArgs.GetDataDirNet() / path;
-    }
+    NodeContext& node = EnsureAnyNodeContext(request.context);
+    fs::path path{AbsPathForConfigVal(EnsureArgsman(node), fs::u8path(request.params[0].get_str()))};
     FILE* file{fsbridge::fopen(path, "rb")};
     CAutoFile afile{file, SER_DISK, CLIENT_VERSION};
     if (afile.IsNull()) {
@@ -2739,7 +2737,6 @@ static RPCHelpMan loadtxoutset()
     LogPrintf("[snapshot] waiting to see blockheader %s in headers chain before snapshot activation\n",
         base_blockhash.ToString());
 
-    NodeContext& node = EnsureAnyNodeContext(request.context);
     ChainstateManager& chainman = *node.chainman;
 
     while (max_secs_to_wait_for_headers > 0) {

```